chains/evm/cli/cli.go

```diff
@@ -52,6 +61,62 @@ func BindEVMCLIFlags(evmRootCLI *cobra.Command) {
 
 }
 
+func writeCliIODataToFile(cmd *cobra.Command) {
+	file, err := os.OpenFile(cliLogsFilename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		log.Error().Err(fmt.Errorf("failed to create cli log file: %v", err))
+	}
+
+	defer file.Close()
+
+	currentTimestamp := time.Now().Format("02-01|15:00.000 ")
+
+	urlFlagName := cmd.Flag(UrlFlagName).Name
```

Comment:
i seemse that you are parsing only global flags, but what will happen when i will call for example:
```./chainbridge-core-example \
evm-cli \
bridge \
register-resource \
--url $NODE_ENDPOINT_NETWORKA \
--privateKey BRIDGE_ADMIN_PRIVATE_KEY \
--bridge 0x0 \
--handler 0x0 \
--target 0x0 \
--resourceId 000000000000000000000000000000e389d61c11e5fe32ec1735b3cd38c69500```
I will not see the exact name of CLI that was called (register-resource) as well as there will not be any CMD specific args